Changes

```diff
@@ -613,6 +613,11 @@ Working version
 * GPR#2173: removed TypedtreeMap
   (Thomas Refis, review by Gabriel Scherer)
 
+- GPR#2188: improvement of the unboxability check for types with a single
+  constructor. Mutually-recursive type declarations can now contain unboxed
+  types. This is based on the paper https://arxiv.org/abs/1811.02300.
+  (Gabriel Scherer and Rodolphe Lepigre, review by ?)
```

Comment:
The changes entry should mention [Mantis 7364](https://caml.inria.fr/mantis/view.php?id=7364).